code/datums/gamemodes/campaign/campaign_rewards.dm

```diff
@@ -62,6 +62,13 @@
 	faction = null
 	return ..()
 
+///Reapplies the asset, refreshing it or restoring uses
+/datum/campaign_reward/proc/reapply()
+	uses += initial(uses)
```

Comment:
Just to be clear, you want behavior here to be carrying over unspent uses rather than strictly refreshing? I'm not particularly familiar with campaign flow.

Reply:
yep. You can stack CAS uses in a single mission, for example.